### DISCUSSION TITLE
Old skool soldering

### DISCUSSION THREAD

**Discussion by CelloGit** [probonopd/MiniDexed#354] [Show and tell]
Built one!
Old skool soldering on standard prototype hat.
Used DAC (great sound!) and separate 10K potmeters for character and backlight brightness.

![minidexed_20220918_143041](https://user-images.githubusercontent.com/106472073/190902736-77d09dc3-870a-4e9c-9e16-564a7412201d.jpg)
![minidexed_20220916_092922](https://user-images.githubusercontent.com/106472073/190902739-c90874df-b7b4-4d79-88eb-87e1e892a448.jpg)
![minidexed_20220918_111917](https://user-images.githubusercontent.com/106472073/190902740-75f25263-01c7-4b15-bf35-8c700ab91fa0.jpg)
![minidexed_20220918_111940](https://user-images.githubusercontent.com/106472073/190902741-6d49630a-1fea-4b06-b2ff-34e68ef5e896.jpg)
![minidexed_20220918_111958](https://user-images.githubusercontent.com/106472073/190902742-d3b06427-f5ac-4756-b268-f56535d055e3.jpg)
![minidexed_20220918_112057](https://user-images.githubusercontent.com/106472073/190902744-162e7ca6-e2c4-487a-8b0d-7613f4bf0388.jpg)

**Comment by diyelectromusic** [comment]
That looks great :)
